src/main/kotlin/com/devfloor/untitled/articleviewcount/domain/ArticleViewCount.kt

```diff
@@ -0,0 +1,30 @@
+package com.devfloor.untitled.articleviewcount.domain
+
+import com.devfloor.untitled.article.domain.Article
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+
+@Entity
+class ArticleViewCount(
+    article: Article,
+) {
```

Comment:
```suggestion
@Entity
@Table(name = "article_view_counts")
class ArticleViewCount(
    article: Article,
) {
```

테이블 어노테이션이 저렇게 쓰는게 맞았나? 잘 기억은 안나는데 다른 엔티티 참고해서 저것도 해두면 좋을 듯!!

Reply:
확인해볼게~